.github/workflows/blueprint.yml

```diff
@@ -1,7 +1,29 @@
+name: Compile blueprint
+
 on:
   push:
     branches:
       - main
+    paths:
+      - '**/*.lean'
+      - '**/blueprint.yml'
+      - 'blueprint/**'
+      - 'docs/**'
+      - 'lean-toolchain'
+      - 'lakefile.toml'
+      - 'lake-manifest.json'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**/*.lean'
+      - '**/blueprint.yml'
+      - 'blueprint/**'
+      - 'docs/**'
+      - 'lean-toolchain'
+      - 'lakefile.toml'
+      - 'lake-manifest.json'
```

Comment:
Does this exclude anything?

Reply:
Yes, everything not lean or blueprint related.